Automattic/wp-calypso
# A4A: Add Partner directory onboarding card.

This pull request  adds the Partner directory onboarding card to the Overview page.
<img width="1720" alt="Screenshot 2024-06-18 at 10 33 39 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/2dee4c4d-115f-462b-8bb3-b8c71f610159">

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/549

## Proposed Changes

* Implement a reusable Sticky card for future usage.
* Implement the Partner directory onboarding card based on the Sticky card component. Use the `a4a-partner-directory-onboarding-card` preference as the flag to display the onboarding card.

## Testing Instructions

* Use the A4A live link and go to the `/overview` page
* Confirm that the onboarding card is visible. Click any of the buttons and confirm the Onboarding card will no longer display.
   * Dismiss button
   * The **Maybe Later** button
   * The **Let's do it** button
* To reset the flow, use the Calypso debug tool at the bottom right corner of the screen and remove `a4a-partner-directory-onboarding-card` preference.

## Pre-merge Checklist
